search/searcher/search_term_prefix.go

```diff
@@ -55,6 +62,8 @@ func NewTermPrefixSearcher(indexReader index.IndexReader, prefix string, field s
 		return nil, err
 	}
 
+	qsearchers = nil
```

Comment:
qsearchers is referenced by the defer'ed cleanup closure.  Setting it to nil makes the cleanup closure not do anything.  (Or, at least that was what I was hoping would happen.)

In some other code, I see there's a qsearchersClose() helper func that's used as the idiom.  I can switch to use that idiom if that makes more sense for readability?